content/src/content/jcr_root/apps/core/wcm/components/image/v1/image/_cq_dialog/.content.xml

```diff
@@ -59,6 +59,20 @@
                                 name="./linkURL"
                                 rootPath="/content"
                                 wrapperClass="cmp-image--editor-link"/>
+                            <linkTarget
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                fieldDescription="Open image link in a new tab."
```

Comment:
`When checked, indicates that the image link should be opened in a new browser tab.`

Reply:
Done